### ISSUE TITLE
Add term tear drop shaped epiphysis

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10377]
**Preferred term label:**
Tear drop shaped epiphysis

**Synonyms**
Teardrop shaped epiphysis, tear drop shaped epiphyses

**Definition (free text, please give PubMed ID)**
Epiphysis with a globular form at the bottom, tapering to a point at the top

**Parent term (use hpo.jax.org/app)**
Abnormal epiphysis morphology HP:0005930

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ACROCAPITOFEMORAL DYSPLASIA; ACFD, OMIM607778

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw @azankl
We try not to include terms that represent subjective descriptions used in only one or two papers. Is this actually a form of cone-shaped epiphysis, and should we use that for the annotation or is this really a characteristic and clearly distinct finding?

**Comment by azankl** [comment]
It's a very distinct finding. It's a key diagnostic feature for Acrocapitofemoral dysplasia [PMID:12624140](https://pubmed.ncbi.nlm.nih.gov/12624140/). But it should be renamed to `cone shaped epiphysis of the femoral head` with a synonym of `tear drop shaped epiphysis of the femoral head` and it should be placed under `Abnormality of the epiphysis of the femoral head HP:0010574`. Here is a picture (from the paper above), arrow points to the cone shaped epiphysis of the femoral head:
<img width="542" alt="Screenshot 2024-03-08 at 10 18 23 am" src="https://github.com/obophenotype/human-phenotype-ontology/assets/4812335/67f9d3c0-6d90-44b2-957b-29eb0187e86c">

**Comment by azankl** [comment]
P.S.: and the term (if accepted) should be assigned to `Acrocapitofemoral dysplasia OMIM:607778`, it's currently the only disorder known to have this feature (so almost pathognomonic)

**Comment by pnrobinson** [comment]
@azankl 

Is this a synonym for 
[Cone-shaped capital femoral epiphysis HP:0008789](https://hpo.jax.org/browse/term/HP:0008789)
The annotation is already used for Acrocapitofemoral dysplasia

**Comment by azankl** [comment]
Yes, its actually a synonym for Cone-shaped capital femoral epiphysis
HP:0008789 <https://hpo.jax.org/browse/term/HP:0008789>, can we just add
this as a synonym?
Thanks
Andreas

**Comment by pnrobinson** [comment]
added synonym "Tear drop-shaped capital femoral epiphysis"
